imsv-docs-astro/src/content/docs/guides/kyc/partner-conducted-kyc.mdoc

```diff
@@ -106,9 +106,7 @@ curl -X POST "https://${imsv_api_host}/api/accounts/${cardholder_account_id}/exp
 {% endpointref name="submit-kyc-statement" /%}
 
 {% note %}
-When testing, to improve chances of a successful response, please provide "passall" as the middleName, and generate a
-random givenName, familyName, and dateOfBirth.
-See example below.
+Read the guidance for testing with Partner Conducted KYC in our guide {% link page="guides/pass-kyc-in-testmode" /%}.
```

Comment:
Could use a similar sentence to my suggestion above, to avoid double 'guide/guidance':
"For a complete guide on passing KYC in test mode, please read {% link page="guides/pass-kyc-in-testmode" /%}.